### ISSUE TITLE
Some naming violates "Go Code Review Comments (Initialisms)"

### ISSUE THREAD

**Issue by dqn** [go-gorp/gorp#419]
This issue is a suggestion in line with golang's standard coding style.Resolving this issue will break backward compatibility and should be considered carefully.

[CodeReviewComments · golang/go Wiki](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

For example, this naming is strange to me.
https://github.com/go-gorp/gorp/blob/master/db.go#L34

**Comment by nelsam** [comment]
Yup.  We can't change them without breaking our public API, and if we're going to do that, we might as well do the full rewrite we were planning way back when.

If you would like to start work on that, I'd be happy to chat about designs in gitter.  Long story short: the next `gorp` (if we even continue to use this repo for it) needs to separate concerns much more than it does currently.  But none of the maintainers use gorp in our daily life, so supporting it is _weird_.  It's much harder to maintain a library when you don't use it (for dozens of reasons that I don't need to get in to here).

**Comment by dqn** [comment]
Thank you for replying.

As you say, the resolving this issue will be in the next `gorp`. When that time comes, consider the issue again.
